tools/utils/seed/template_locals.ts

```diff
@@ -1,21 +1,46 @@
+import * as util from 'gulp-util';
 import { argv } from 'yargs';
+import { join } from 'path';
+
 import * as CONFIG from '../../config';
 
+const getConfig = (path: string, env: string): any => {
+  const suffixPath = join(path, `${env}.config.js`);
+  const simplePath = join(path, `${env}.js`);
+  const config: any = [suffixPath, simplePath]
+    .reduce((config: any, path: string) => {
+      if (config) {
+        return config;
+      }
+      try {
+        config = require(path);
```

Comment:
We can remove the extension from the require and it should work. I'll verify later today.